app/src/main/java/com/github/pedrovgs/effectiveandroidui/domain/tvshow/Catalog.java

```diff
@@ -34,6 +34,7 @@
    * Default constructor. All this hardcoded information is going to be used as mocked information
    * for the demo application.
    */
+  @SuppressWarnings("PMD.AvoidDuplicateLiterals")
   public Catalog() {
     this.tvShows = new LinkedHashSet<TvShow>();
     TvShow tvShow =
```

Comment:
Once again, I treat this rule as an special case. PMD complains when it sees a String literal repeated often, as it could be written as a private constant.

In this class, "Pilot" appears 12 times. But as this is a sample app, and these data is harcoded for sample purposes, it makes sense to waive this rule.

In a realistic scenario, this rule will complain when finding the same string 5 times or more, with 3 as a minimum length: http://pmd.sourceforge.net/pmd-5.1.1/rules/java/strings.html#AvoidDuplicateLiterals